client/dashboard/sites/settings-subscription-gifting/summary.tsx

```diff
@@ -0,0 +1,26 @@
+import { __ } from '@wordpress/i18n';
+import RouterLinkSummaryButton from '../../components/router-link-summary-button';
+import { SiteSettings } from '../../data/types';
+
+interface SubscriptionGiftingSettingsSummaryProps {
+	siteSlug: string;
+	settings: SiteSettings;
+}
+
+export default function SubscriptionGiftingSettingsSummary( {
+	siteSlug,
+	settings: { wpcom_gifting_subscription },
+}: SubscriptionGiftingSettingsSummaryProps ) {
+	return (
+		<RouterLinkSummaryButton
```

Comment:
This doesn't appear to have a focus state when I keyboard through on Chrome. Is that something to fix in this PR?

Reply:
It looks to be a DS issue, we can solve it later I think, but I'll note! 🤔

Reply:
Filed an issue here: DES-120 🙇 thanks for catching this!